pandas/tseries/tdi.py

```diff
@@ -353,36 +353,16 @@ def days(self):
         return self._get_field('days')
 
     @property
-    def hours(self):
-        """ The number of integer hours for each element """
-        return self._get_field('hours')
-
-    @property
-    def minutes(self):
-        """ The number of integer minutes for each element """
-        return self._get_field('minutes')
-
-    @property
     def seconds(self):
-        """ The number of integer seconds for each element """
+        """ The number of integer total seconds for each element """
         return self._get_field('seconds')
 
     @property
-    def milliseconds(self):
-        """ The number of integer milliseconds for each element """
-        return self._get_field('milliseconds')
-
-    @property
     def microseconds(self):
         """ The number of integer microseconds for each element """
         return self._get_field('microseconds')
 
     @property
-    def nanoseconds(self):
-        """ The number of integer nanoseconds for each element """
```

Comment:
Should we keep this one? As this is not available in `datetime.timedelta`, and it completes the row of day-second-microsecond to have your full timedelta value expressed in those parts. 
But maybe that is making it just complicated ..